rc/base/lint.kak

```diff
@@ -87,13 +87,13 @@ define-command -hidden lint-show-counters %{
 }
 
 define-command lint-enable -docstring "Activate automatic diagnostics of the code" %{
-    add-highlighter window flag_lines default lint_flags
+    add-highlighter window/lint-highlighter flag_lines default lint_flags
```

Comment:
why all those `-highilghter` in highlighter names ? from the context we already know those are highlighters, so it seems unnecessary to add that in the name as well. I'd prefer just `lint` here, `git-blame` there...

Reply:
I saw in `git-tools` the existing highlighters have this prefix, I thought it's a pattern that you introduced and I wanted to keep the consistency 🙂

add-highlighter window/git-log-highlight
add-highlighter window/git-status-highlight

Do you want me to drop the `-highlight` suffix from everywhere?

Reply:
There's a commit now that removes the suffix in the files I touched